Libraries/Utilities/differ/transformsDiffer.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+'use strict';
+
+const transformsDiffer = function(one, two) {
```

Comment:
if both one and two are null, this would return true right? that seems not totally correct